packages/infrastructure/ore-client/src/helper/getDOMTOM.ts

```diff
@@ -0,0 +1,13 @@
+import { Option } from '@potentiel-libraries/monads';
+
+import { DOMTOMPostalCodeRanges, RelevantDOMTOM, relevantDOMTOM } from '../constant';
+
+export const getDOMTOM = (codePostal: number): Option.Type<RelevantDOMTOM> => {
+  for (const DOMTOM of relevantDOMTOM) {
+    const range = DOMTOMPostalCodeRanges[DOMTOM];
+    if (codePostal >= range.min && codePostal <= range.max) {
+      return DOMTOM;
+    }
+  }
+  return Option.none;
+};
```

Comment:
juste une suggestion de simplification (selon moi !)
```suggestion
const domToms = {
  "971": "Guadeloupe"
}

export const getDOMTOM = (codePostal: string): Option.Type<RelevantDOMTOM> => {
  return domToms(codePostal.slice(0,3)) ?? return Option.none;
};
```

j'ai remis en string car la data est stockée en string, mais ca marche aussi bien en number, as you wish

Reply:
Ok np ça